base/operators.jl

```diff
@@ -827,17 +827,25 @@ julia> [1:5;] |> x->x.^2 |> sum |> inv
 Compose functions: i.e. `(f ∘ g)(args...)` means `f(g(args...))`. The `∘` symbol can be
 entered in the Julia REPL (and most editors, appropriately configured) by typing `\\circ<tab>`.
 
+Function composition also works in prefix form: `∘(f,g)` is the same as `f ∘ g`.
+The prefix form also supports multiple function composition: `∘(f,g,h) = f ∘ g ∘ h`
```

Comment:
I'm not a native speaker, but how does "supports composition of multiple functions" sound?

Reply:
```suggestion
The prefix form also supports composition of multiple functions: `∘(f,g,h) = f ∘ g ∘ h`
```